hasher-matcher-actioner/src/OpenMediaMatch/blueprints/curation.py

```diff
@@ -89,6 +93,10 @@ def bank_update(bank_name: str):
         storage.bank_update(bank, rename_from=rename_from)
     except ValueError as e:
         abort(400, *e.args)
+    except IntegrityError:
+        abort(403, "Bank already exists")
```

Comment:
blocking: This is an update function, the bank is expected to exist (otherwise it will 404 on L82). In what circumstances are IntegrityError thrown?

Reply:
I believe same as in create, where the passed in `name` already exists (name is set to `unique=True` in db)